README.md

```diff
@@ -4,73 +4,88 @@ This repository contains reproducible benchmarks to evaluate the instrumentation
 
 The focus is on Performance Monitoring (tracing) of web servers.
 
-## Usage
-
-You will need `docker` and `go`.
+## How It Works
 
-```shell
-go install .
 ```
-
-```shell
-sentry-sdk-benchmark platform/python/django
+                      ┌────────────────────┐
+                      │┼┼┼┼┼┼┼┼┼┼┼┼┼┼┼┼┼┼┼┼│
+   ┌───────────┐      │┼┼────────────────┼┼┼┐      ┌────────┐
+   │           │      │┼│                │┼┼│      │        │
+   │ Benchmark ├─────►├┼│ Docker Compose │┼┼┼─────►│  HTML  │
+   │  Runner   │      │┼│    Projects    │┼┼│      │ Report │
+   │           │      │┼│                │┼┼│      │        │
+   └──────────┬┘      │┼┼────────────────┼┼┼│      └────────┘
+              │       └┼┼┼┼┼┼┼┼┼┼┼┼┼┼┼┼┼┼┼┼┼│
+              │        └────────────────────┘
+              │
+       ┌──────┴────────────────────────┐
+       │ Selection of prepackaged apps │
+       │   instrumented with Sentry    │
+       └───────────────────────────────┘
 ```
 
-## How It Works
-
-The `sentry-sdk-benchmark` command automates the following steps:
+We provide a [collection of prepackaged web apps](platform/) implemented in different programming languages and using different frameworks.
 
-1. Start `baseline` app:
+For each original app in a "baseline" directory, there is a slightly modified app in a corresponding "instrumented" directory which is a copy of the original app with minimal changes to add Sentry instrumentation (error and performance monitoring).
```

Comment:
```suggestion
For each original app in a "baseline" directory, there is a slightly modified app in a corresponding "instrumented" directory which is a copy of the original app with minimal changes to add Sentry instrumentation (error and performance monitoring). The "baseline" apps are implementations from [TechEmpower Framework Benchmarks](https://github.com/TechEmpower/FrameworkBenchmarks).
```

Reply:
Your feedback makes me think we have a blurry line right now between implementation detail and general design.

I thought of this section of the main `README.md` as a high level overview to help understand what the project is about. And in #27, in `platform/README.md`, detail the current implementation constraints.

Let's have a quick chat to see if we can delineate the separation better.

Reply:
We decided to embrace the current implementation constraints and leave generalizations for the future.